README.md

```diff
@@ -0,0 +1,39 @@
+# Comrade Colonel Bot
+
+Telegram bot removing group chat messages after specified period of time.
+
+To try it quickly, run:
+
+```sh
+export TELEGRAM_BOT_TOKEN=123456789:ABCDEFGHIJKLMNOPQRSTUVWXYZ123456789
+export MESSAGE_LIFETIME=5
+export DELETION_PERIOD=1
+cargo run
+```
+
+Don't forget to specify your actual Telegram bot token (you're able to get one from `@BotFather`). The bot will be configured to delete all messages he has evidenced after 5 seconds. [Maximal possible](https://core.telegram.org/bots/api#deletemessage) manageable chat message lifetime is 48 hours (172800 seconds).
+
+## Deployment
+
+1. Install Kubernetes,
+
+1. Create Kubernetes' secret containing Telegram bot token:
+
+   ```sh
+   echo -n '123456789:ABCDEFGHIJKLMNOPQRSTUVWXYZ123456789' > ./telegram-bot-token
+   kubectl create secret generic comrade-colonel-bot --from-file=./telegram-bot-token
+   ```
+
+1. Fetch bot's deployment configuration file:
+
+   ```sh
+   wget https://raw.githubusercontent.com/loyd/comrade-colonel-bot/master/k8s-deploy-comrade-colonel-bot.yml
```

Comment:
I believe that duplicate `comrade-colonel-bot` once again isn't necessary:
```sh
whet https://raw.githubusercontent.com/loyd/comrade-colonel-bot/master/k8s-deploy.yml | kubectl -f -
```

So, we should call the config below "k8s-deploy.yml"